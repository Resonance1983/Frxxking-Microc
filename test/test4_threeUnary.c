void main(){
    int i=2;
    (i > 1) ? print(1):print(2);
}