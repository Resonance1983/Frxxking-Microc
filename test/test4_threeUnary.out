24 19 0 5 25 15 1 13 0 0 1 0 2 12 15 -1 13 0 0 1 11 0 1 10 7 17 32 0 1 22 16 35 0 2 22 15 -1 15 -1 21 -1